modules/ROOT/pages/_partials/dataweave-formats-flatfile-config.adoc

```diff
@@ -82,6 +82,7 @@ Valid values are `true` or `false`.
 Note that in Mule versions 4.0.4 and later, this is only used as a separator
 when there are multiple records. Values translate directly to character codes
 (`none` leaves no termination on each record).
+| `retainEmptyStringFieldsOnParsing` | `false` | When set to `true`, this property makes the reader retain fields that lack values and set their values to an empty string. _Introduced in the January 2023 version of DataWeave 2.3 for Mule version 4.3.0-20221212._
```

Comment:
The property impacts only for String fields. May be we could clarify it.

Reply:
@sms-asalanova I need a commit suggestion please.

Reply:
That also needs to be reconciled with the approved release note. Please assist for the release.

Reply:
We've decided to improve this later if necessary. Feel free to ping me about the text you want.